### PR TITLE
[4.0] scss variable names

### PR DESCRIPTION
**Pull request by brianteeman**
The new template introduced some new variables for mobile. They are in german so this simple pr changes them to english.

### CODE REVIEW

File: administrator/templates/atum/scss/_variables.scss [hunk 1]
```diff
@@ -258,6 +258,6 @@ $zindex-modal-backdrop:            1040;
 $zindex-modal:                     1050;
 $zindex-popover:                   1060;
 $zindex-tooltip:                   1070;
-$zindex-mobil-bottom:              8000;
-$zindex-mobil-toggle:              9999;
-$zindex-mobil-menu:                9000;
+$zindex-mobile-bottom:              8000;
```
Comment (Quy) [inline administrator/templates/atum/scss/_variables.scss]: Align second column

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7bd78391d1a6761e48c2c8b083d8d967be512d18<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26118">issues.joomla.org/tracker/joomla-cms/26118</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**